src/Workspaces/VisualBasicTest/CodeGeneration/AddImportsTests.vb

```diff
@@ -973,7 +973,7 @@ End Class", useSymbolAnnotations:=True)
         'End Class", useSymbolAnnotations)
 
         '            Dim doc = Await GetDocument(source, useSymbolAnnotations)
-        '            Dim options As OptionSet = Await doc.GetOptionsAsync()
+        '            Dim options As Options = Await doc.GetOptionsAsync()
```

Comment:
What's up with this test? It's just commented out?...

Reply:
IDK :(